### ISSUE TITLE
Documentation not clear on using the file manager with as a picker for text fields

### ISSUE THREAD

**Issue by Elendurwen** [trippo/ResponsiveFilemanager#292]
Hi, I would like to use the file manager as a popup window that opens when you click a "Pick an image" button next to a text field. Once user clicks an image, the popup closes and the text field with a certain ID gets the URL filled in.

I tried following the documentation that you have on the web site and I also looked at the code you have on the demo page but I can't get this working. Also, none of the "Select" buttons on the demo page actually work for me, nothing happens when I click them. I tried on Firefox and Safari.

Could you please make the documentation more clear on how I can achieve this? Or let me know in the comments?

PS: I have the file manager already working with tinymce so it is already setup properly.

Thanks!

**Comment by trippo** [comment]
It's very strange because for me work very well, can i see a online demo of your page?

**Comment by Elendurwen** [comment]
The demo worked for me on a different computer which is strange.. I wasn't able to get the popup working with a fancy box though, the modal background wasn't closing when you selected an image. In the end, I implemented my own modal window and it worked ok. 

You might want to check whether it's working on with the newest fancy box, I saw that on the demo you were using an older version. Also, better documentation would be great for the future!

**Comment by danMontague** [comment]
Yeah, I have managed to get the filemanager to open when clicking a button next to text input. Selecting an image populates the field but does not close the filemanager window and the documentation is not clear on this. Can this be looked at or someone provide some advice please

**Comment by cdaveb** [comment]
danMontague- if you're using Fancybox 3.x for the popup window, the current version of Responsive File manager won't properly close the window. If this is your issue, there's a fix I posted in issue ticket: #345

**Comment by danMontague** [comment]
@cdaveb. Thanks for that. Just what I was after.
